lib/src/spec/validationEngine.spec.ts

```diff
@@ -6,168 +6,198 @@ import { FieldValidationResult } from '../entities';
 
 //TODO: Implement Issue #20
```

Comment:
I think this issue number comes from bitbucket private repository. Maybe we need new issue to update TODOs